src/NodaTime.Web/Markdown/unstable/installation.md

```diff
@@ -0,0 +1,50 @@
+@Title="Installation"
+
+Our primary distribution channel is [NuGet](https://www.nuget.org/) with
+three packages:
+
+- [NodaTime](https://www.nuget.org/packages/NodaTime)
+- [NodaTime.Testing](https://www.nuget.org/packages/NodaTime.Testing)
+- [NodaTime.Serialization.JsonNet](https://www.nuget.org/packages/NodaTime.Serialization.JsonNet)
+
+Alternatively, source and binary downloads are available on the
+[project download page][downloads].
+
+[downloads]: /downloads/
+
+See the ["Building and testing"][building] section in the developer guide for
+instructions on building Noda Time from source.
+
+[building]: /developer/building
+
+System requirements
```

Comment:
This is the only section I've changed here. (Is there some way to force a diff against `src/NodaTime.Web/Markdown/2.0.x/installation.md`?)

Reply:
Within GitHub? Not that I'm aware of. I could do it locally, but I trust you :)